taxi/migrations/0003_alter_manufacturer_name.py

```diff
@@ -6,13 +6,13 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('taxi', '0002_alter_manufacturer_options_and_more'),
+        ("taxi", "0002_alter_manufacturer_options_and_more"),
     ]
 
     operations = [
         migrations.AlterField(
-            model_name='manufacturer',
-            name='name',
+            model_name="manufacturer",
+            name="name",
             field=models.CharField(max_length=255, unique=True),
```

Comment:
The `name` field in the `Manufacturer` model is now unique. Ensure this change is reflected in your `models.py` to maintain consistency between your models and migrations.